x/pool-incentives/keeper/hooks.go

```diff
@@ -22,3 +22,28 @@ func (h Hooks) AfterPoolCreated(ctx sdk.Context, poolId uint64) {
 		panic(err)
 	}
 }
+
+// execute distribution after adding collected fees to fee pool
+func (h Hooks) AfterAddCollectedFees(ctx sdk.Context, fees sdk.Coins) {
+	// @Sunny, @Tony, @Dev, what comments should we keep after modifying own BeginBlocker to hooks?
+
+	// WARNING: The order of how modules interact with the default distribution module matters if the distribution module is used in a similar way to:
+	// 1. mint module or 2. custom mint modules that uses the auth module’s feeCollector to mint new tokens that uses the default distribution module.
+	// Currently, the mint module mints inflation amount to the feeCollector module account,
+	// and on the next BeginBlock the distribution module uses all the available balance of the feeCollector to process the distribution.
+	// Therefore, for the pool-incentives module to only take the AllocationRatio from the inflated amount, it should be run before the distribution module’s BeginBlock.
```

Comment:
What is this `AllocationRatio`?

Reply:
Minted token distributions are implemented as we discussed before